h2o-r/h2o-package/R/frame.R

```diff
@@ -2428,8 +2414,7 @@ h2o.ascharacter <- function(x) {
 #' @param ... Further arguments to be passed from or to other methods.
 #' @export
 h2o.print <- function(x,n=6L) {
-  if(is.H2OFrame(x)) print(x,n)
-  else print(x,n)
+  print(x,n)
```

Comment:
`n` is passed to `...` generic print, then matched to `n` in h2o print method, no issue here, but name `n` explicitly: `print(x, n = n)` so argument passed to dots won't rely on position.